praw/objector.py

```diff
@@ -1,10 +1,29 @@
 """Provides the Objector class."""
+import re
+
 from .exceptions import APIException
 
 
 class Objector(object):
     """The objector builds :class:`.RedditBase` objects."""
 
+    @staticmethod
+    def camel_to_snake(name):
+        """Convert camelCase string to snake_case."""
```

Comment:
Can you also please add a reference to the stack overflow answer that lead to this solution in the docstring?